eng/scripts/Update-PkgVersion.ps1

```diff
@@ -91,9 +91,15 @@ if ($packageSemVer.HasValidPrereleaseLabel() -ne $true){
 }
 
 if (!$packageOldSemVer.IsPrerelease -and ($packageVersion -ne $NewVersionString)) {
+  $whitespace = $propertyGroup["Version"].PreviousSibling
   if (!$propertyGroup.ApiCompatVersion) {
     $propertyGroup.InsertAfter($csproj.CreateElement("ApiCompatVersion"), $propertyGroup["Version"]) | Out-Null
-    $whitespace = $propertyGroup["Version"].PreviousSibling
+    $propertyGroup.InsertAfter($whitespace.Clone(), $propertyGroup["Version"]) | Out-Null
+  }
+  $ApiCompatVersionComment = "The ApiCompatVersion will be updated automatically and does not need to be modified."
```

Comment:
Maybe we should just always replace in case we want to change the comment in the future.